modules/wpaperd/hm.nix

```diff
@@ -1,35 +1,39 @@
 { config, lib, ... }:
-
+let
+  cfg = config.stylix.targets.wpaperd;
+in
 {
-  options.stylix.targets.wpaperd.enable =
-    config.lib.stylix.mkEnableTarget "wpaperd" true;
+  options.stylix.targets.wpaperd = {
+    enable = config.lib.stylix.mkEnableTarget "wpaperd" true;
+    useWallpaper = config.lib.stylix.mkEnableWallpaper "wpaperd" true;
+  };
```

Comment:
Same here - `wpaperd`'s only purpose is to set the wallpaper so I don't see a benefit of `useWallpaper` and `enable` being separate options.